cpp/ps0008/problem. 8.cpp

```diff
@@ -38,11 +44,16 @@ int main() {
         } else {
             std::cout << "Wrong input, try again" << std::endl;
         }
-        }
+    }
+    
     std::cout << "Enter the encrypted word or sentence:\n  ";
-    std::cin.ignore(); 
-    std::getline(std::cin, input);
+    std::cin.ignore();
+    std::getline(std::cin, input); // Taking message 1/1
+    
+    // invoke your decrypt function to return the plaintext 1/1
     std::string decryptedText = decrypt(input, key);
     std::cout << "Decrypted text: " << decryptedText << std::endl;
+    
     return 0;
-}
\ No newline at end of file
+}
+// great work 😎 9/10 ⭐
```

Comment:
- [x] readable code with comments explains most of its blocks
- [x] did a commit in a right folder for your GitHub repo
- [x] used `string`, `getline` without any problems
- [ ] no bugs while execution and prompting
+ 0.5 bonus mark for using references ⭐

try to give a look at my [source code](https://onlinegdb.com/o_VOTbbro)